File: JAVA/IDADES.java
package curso_programacao;
import java.util.Locale;
import java.util.Scanner;

public class IDADES {
	public static void main(String[] args) {
		
		Locale.setDefault(Locale.US);	
		Scanner sc = new Scanner(System.in);
	
		int idade1, idade2;
		double media;
		String nome1, nome2;
	System.out.println("Dados da primeira pessoa: ");
	System.out.print("Nome: ");
	nome1 = sc.nextLine();
	System.out.print("Idade: ");
	idade1 = sc.nextInt();
	
	System.out.println("Dados da segunda pessoa: ");
	System.out.print("Nome: ");
	nome2 = sc.nextLine();
	sc.nextLine();
	System.out.print("Idade: ");
	idade2 = sc.nextInt();
	
	media = (idade1 + idade2) / 2.0;
	
	System.out.print("A idade media de " + nome1 + " e " + nome2 + " eh de " + media + " anos.");
	
	
	
	
	
		
		
		
	sc.close();
	}
}

File: Cpp/IDADES.cpp
#include <bits/stdc++.h>

using namespace std;

int main()
{

    int idade1, idade2;
    double media;
    string nome1, nome2;

    cout << "Dados da primeira pessoa: " << endl;
    cout << "Nome: ";
    getline(cin ,nome1);
    cout << "Idade: ";
    cin >> idade1;
    cout << "Dados da segunda pessoa: " << endl;
    cin.ignore(INT_MAX, '\n');
    cout << "Nome: ";
    getline(cin, nome2);
    cout << "Idade: ";
    cin >> idade2;

    media = (idade1 + idade2) / 2.0;

    cout << fixed << setprecision(1);
    cout << "A idade media de " << nome1 << " e " << nome2 << " eh de " << media << " anos. " << endl;



    return 0;
}
